python/lsst/ap/association/diaPipe.py

```diff
@@ -647,16 +648,21 @@ def run(self,
                     ["diaObjectId", "diaForcedSourceId"],
                     drop=False,
                     inplace=True)
+
             self.alertPackager.run(assocResults.associatedDiaSources,
                                    diaCalResult.diaObjectCat,
                                    preloadedDiaSources,
                                    diaForcedSourcesFull,
                                    diffIm,
                                    exposure,
                                    template,
+                                   ssSrc=associatedSsSources,
                                    doRunForcedMeasurement=self.config.doRunForcedMeasurement,
                                    forcedSourceHistoryThreshold=forcedSourceHistoryThreshold,
                                    )
+        if associatedSsSources is not None:
```

Comment:
I assume you're doing this so the MPCORB columns aren't stored in the butler datatype?  Maybe add a comment to that effect.